chatbot-core/backend/app/settings/__init__.py

```diff
@@ -0,0 +1,8 @@
+from .constants import Constants
+from .secrets import Secrets
+
+
+__all__ = [
+    "Constants",
+    "Secrets",
+]
```

Comment:
What is this used for ?

Reply:
> What is this used for ?
This means the setting module will return two exports `Constant` and `Secrets`.
Only when used with `import *` tho, which is not recommended.

Reply:
As @lekiet1214 said, I saw commonly from LlamaIndex library, so I just copy that.

Reply:
Ok good.